src/support/slack/slack-handler.js

```diff
@@ -46,6 +46,7 @@ function SlackHandler(commands, log) {
       threadTs,
       client,
       user: event?.user,
+      botToken: context.botToken || process.env.SLACK_BOT_TOKEN,
```

Comment:
we don't have `process.env` on the lambda so to speak. are you looking maybe for `context.env`  (the lambda context)?